tests/components/zwave_js/fixtures/bulb_6_multi_color_state.json

```diff
@@ -591,7 +591,7 @@
         "type": "any",
         "readable": true,
         "writeable": false,
-        "label": "Libary type"
+        "label": "Library type"
```

Comment:
This data comes from the driver or device. I don't think we should fix these typos as this is fixture data that should represent the data as it is received from the driver or device.